### PR TITLE
repokitteh: add warning for draft PRs

### PR DESCRIPTION
**Pull request by alyssawilk**
adding a warning for draft PRs.

### CODE REVIEW

File: ci/repokitteh/modules/newpr.star [hunk 1]
```diff
@@ -8,26 +8,52 @@ In the meantime, please take a look at the [contribution guidelines](https://git
...
+    action == 'opened'
+    and is_draft(issue_number))
+
 def send_newcontributor_message(sender):
   github.issue_create_comment(NEW_CONTRIBUTOR_MESSAGE % sender)
 
+def send_draft_wip_notice(sender):
+  github.issue_create_comment(DRAFT_MESSAGE)
+
 def _pr(action, issue_number, sender, config):
```
Comment (itayd) [inline ci/repokitteh/modules/newpr.star:53]: if you add `draft` to the parameters, it'll be set to `True` if this pr is a draft, no need to get the pr from that.

File: ci/repokitteh/modules/newpr.star [hunk 2]
```diff
@@ -8,26 +8,52 @@ In the meantime, please take a look at the [contribution guidelines](https://git
...
+or be handled by maintainer-oncall triage.
+
+Please mark your PR as ready when you want it to be reviewed!
+"""
+
+
 def get_pr_author_association(issue_number):
   return github.call(
     method="GET",
     path="repos/envoyproxy/envoy/pulls/%s" % issue_number)["json"]["author_association"]
```
Comment (itayd) [inline ci/repokitteh/modules/newpr.star:22]: how does author_association different than author?

**Comment by itayd** [comment]
you can test this by "blessing" the pr (lmk if you don't know what that means) and setting this pr as draft.

File: ci/repokitteh/modules/newpr.star (hunk 2 above)
Comment (alyssawilk) [reply ci/repokitteh/modules/newpr.star]: not sure - this is existing repokitteh code, not from my change :-)

**Comment by alyssawilk** [comment]
blessed and drafted.  no update so I assume something is broken but I don't know where logs go?

**Comment by alyssawilk** [comment]
specifically this script is supposed to run on PR creation so I'm not sure I can use this workflow.  WDYT?

**Comment by itayd** [comment]
ah, if it's on a pr creation, you can bless in the description as well

like,
put /rk-bless when you create a pr.

**Comment by itayd** [comment]
tbh, this is simple enough - you can merge and see if it works right away

**Comment by alyssawilk** [comment]
Gotcha.  FWIW this is literally the same code that you said broke things last time, so I don't think we want to merge again without sorting out what happened :-)

**Pull request closed without merging**